packages/atomic/src/components.d.ts

```diff
@@ -60,12 +60,12 @@ export namespace Components {
         "updateMessage": (region: string, message: string, assertive: boolean) => Promise<void>;
     }
     /**
-     * @beta - This component is part of the automatic facets feature.
-     * Automatic facets are currently in beta testing and should be available soon.
-     * An automatic facet is a special type of facet generated by the automatic facets feature. Unlike traditional facets that
-     * need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
-     * in response to the search query. It's important to note that this component should not be used directly. It is internally used
-     * by the `atomic-automatic-facet-generator` component to automatically render updated facets.
+     * An automatic facet is a special type of facet generated by the automatic facets feature.
+     * Unlike traditional facets that need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
+     * in response to the search query.
+     * It's important to note that this component is not typically used directly.
+     * It is internally used by the `atomic-automatic-facet-generator` component to automatically render updated facets.
+     * To read more about the automatic facet generator feature, see: https://docs.coveo.com/en/3383#about-the-facet-generator-feature
```

Comment:
Ah, here's the docs I was asking about 🙂 I don't feel too strongly, but perhaps this would be best to be referenced in-line where I suggested above? It's fine either way though.